src/pages/Home/components/ModalWindow/MovieAddEdit/MovieAddEdit.jsx

```diff
@@ -1,80 +1,94 @@
-import React, {useEffect, useState} from 'react';
+import React from 'react';
+import {useFormik} from 'formik';
+import * as Yup from 'yup';
+import {useDispatch} from 'react-redux';
 import ModalWindow from 'pages/Home/components/ModalWindow/ModalWindow';
-import InputField from "pages/shared/InputField/InputField";
-import InputSelect from "pages/shared/InputSelect/InputSelect";
+import InputField from 'pages/shared/InputField/InputField';
+import InputSelect from 'pages/shared/InputSelect/InputSelect';
 import 'pages/Home/components/ModalWindow/MovieAddEdit/MovieAddEdit.scss';
-import {useDispatch} from "react-redux";
-import {addMovie, updateMovie} from "store/actions/MoviesActions";
+import {addMovie, updateMovie} from 'store/actions/MoviesActions';
+import {genres} from 'utils/constants';
 
 const MovieAddEdit = ({onClose, movie, isEdit}) => {
 
-    const [form, setForm] = useState({
-        id: '',
+    const dispatch = useDispatch();
+
+    const initObject = isEdit ? {
+        id: movie.id,
+        title: movie.title,
+        release_date: movie.date,
+        genres: movie.genres,
+        overview: movie.overview,
+        runtime: movie.runtime == null ? 0 : movie.runtime,
+        poster_path: movie.poster_path,
+    } : {
         title: '',
-        releaseDate: '',
-        genres: [],
+        release_date: new Date().toISOString().substr(0, 10),
+        genres: genres,
```

Comment:
It seems in this case we have all selected options in "select". Maybe it should be as before genres: [],